src/kbmod/regionsearch/utilities.py

```diff
@@ -0,0 +1,210 @@
+import os
+import random
+import typing
+
+import astropy.table
+import numpy as np
+from astropy import time
+from astropy import units as u
+from astropy.coordinates import EarthLocation, SkyCoord, solar_system_ephemeris
+
+
+class RegionSearchClusterData(object):
+    """Randomly generated cluster data for testing the region search.
```

Comment:
The first version I wrote had a bug and was really really slow. I added all the cache stuff to speed up testing and then found the bug. So generation is faster now but still pretty slow and might be a problem for CI unit tests.  The "tmp" directory is in .gitignore and so not controlled. To persist a dataset so that it isn't generated each time it can be put in a data directory.

Reply:
Thanks for the explanation.